src/components/DataSubmissions/DataUpload.tsx

```diff
@@ -27,7 +27,7 @@ const StyledMetadataText = styled(StyledUploadTypeText)(() => ({
 
 const StyledUploadFilesButton = styled(Button)(() => ({
   minWidth: "137px",
-  minHeight: "44px",
+  height: "44px",
```

Comment:
For the other buttons I just removed the height property and made sure the padding is 10px and lineHeight is 24px to match the Figma. The 1px border will increase the height to 46px. (All other buttons are 46px) Let me know if you agree, or if we should modify the heights of the other buttons to a fixed 44px.

Reply:
Agreed. I reverted the height override so the buttons are all consistent. 66b8644